core/views.py

```diff
@@ -402,7 +408,27 @@ def handle_no_permission(self: "CommunityUpdateView") -> HttpResponse:
         messages.error(self.request, "You do not have permission to update this community.")
         return redirect("community-detail", slug=self.get_object().slug)
 
+    def _handle_image_update(
+        self: "CommunityUpdateView", form: forms.ModelForm, old_instance: models.Model, field_name: str
+    ) -> None:
+        clear_flag = self.request.POST.get(f"{field_name}-clear")
```

Comment:
Could it be moved to processing image in the model itself? Right now we have image processing in two places at least. It will be hard to maintain and debug. Or maybe add class method on community object? Would be good to keep that logic in one place.

Reply:
@maksmondeo please check above comment